packages/affiliates/apps/DeployerRewards.js

```diff
@@ -31,35 +32,41 @@ async function App(config) {
   const client = new BigQuery();
   const queries = Queries({ client });
   const coingecko = Coingecko();
+  const synthPrices = SynthPrices({ web3 });
 
   const rewards = DeployerRewards({
     queries,
-    coingecko,
+    empCreatorAbi,
     empAbi,
-    empCreatorAbi
+    coingecko,
+    synthPrices
   });
 
   // get emp info
-  const { tokensToPrice, tokenDecimals } = await Promise.reduce(
+  const { collateralTokens, collateralTokenDecimals, syntheticTokenDecimals } = await Promise.reduce(
     empWhitelist,
     async (result, address) => {
       // switch this to tokenInfo if you want to base prices off tokens
-      const info = await emp.collateralInfo(address);
-      result.tokensToPrice.push(info.address);
-      result.tokenDecimals.push(info.decimals);
+      const collateralInfo = await emp.collateralInfo(address);
+      result.collateralTokens.push(collateralInfo.address);
+      result.collateralTokenDecimals.push(collateralInfo.decimals);
+
+      const syntheticInfo = await emp.collateralInfo(address);
```

Comment:
this should probably be emp.tokenInfo

Reply:
ye, you 100% right. Update updated accordingly